packages/image_picker/.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should the `.vscode/` folders be `.gitignored` or are they intended to be shared between developers using that IDE?